source/_integrations/reolink.markdown

```diff
@@ -386,11 +386,18 @@ Test if you can access the camera by its IP address in your browser `https://<yo
 
 Set up the Reolink integration in Home Assistant using the credentials you set in step 1.
 
-## Showing the camera in the dashboard
+## Examples
+
+### Showing the camera in the dashboard
 
 1. One way to show the camera in the dashboard is by using the picture glance card.
 2. For example, you can place arrow buttons on the card to [control the camera](/dashboards/picture-glance/#creating-a-card-to-control-the-camera).
 
+### Sending rich notifications
+
+Using Home Assistant you can receive rich notifications on your phone when someone rings a Reolink doorbell or a Reolink camera detects an event like motion or a person.
+[This guide](https://github.com/starkillerOG/reolink_aio/blob/main/doc/Rich%20notifications%20using%20Home%20Assistant.pdf) will show the necessary steps to create the automation.
```

Comment:
I don't think this should be in some kind of PDF hosted on GitHub.

If there is information in there that is relevant for this documentation, we should add it here.

Reply:
It's a step by step guide on how to make the automation for beginners.
I can just include this guide in the HomeAssistant docs instead of linking, but I don't want it directly in the docs since it would make the docs really long and potentialy I would like to add more of these guides in the future.
Its about 5 pages long.

Maybe I could make it a foldable section like the "manual configuration steps", but don't exactly know how that works.
Would that be acceptable?

It could also be on some other page, but I don't think that is allowed and I would also not know where to put it.
Is a seperate page acceptable?

Or I could convert it from the PDF to just a Readme on the reolink_aio github (the upstream libary dependency).
Would that be acceptable?

Reply:
We really think this should just go into the documentation. My original comment thus holds.

../Frenck

Reply:
Alright, will convert it to just plain text and include it directly in the docs.
Will probably do it somewhere next week.